src/Home.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import HerosSec from './component/HerosSec';
+import NavBar from './component/NavBar';
+import HeroSec2 from './component/HeroSec2';
+import books from './assets/back-nav.jpg'
+import abc from './assets/teach-home.jpg'
+import { Link } from 'react-router-dom';
+import './home.css'
+
+const Home = () => {
+
+  const data={
+    start_journey:'Start your journey here',
+    hero_heading:'Tutoring',
+    hero_para1:'Teach others to get points and use that points to get free one on one learning...',
+    hero_para2:'Teaching is another form of learning!!!',
+    image_home:abc,
+    page:'home'
+  }
+  return (
+    <>
+      <header className='home-header'>
+        <NavBar></NavBar>

```

Comment:
Why `NavBar` is used inside `Home.js` when you have already defined it before Routes in `App.js` ?